app/atoms/ln-patternlab/_ln-m-widget-contact.scss

```diff
@@ -0,0 +1,26 @@
+$_phone_prefix: 'T';
```

Comment:
I was thinking of just moving the flat folder to "app" because it will only have style files inside. Something like this:

```
app/
  patternlab/
    _ln-m-widget-contact.scss
    _ln-m-widget-downloads.scss
    ...
    _ln-o-widgets-area.scss
    _ln-o-gravity-form.scss
    ...
  atoms/
  molecules/
  organisms/
  templates/
````

But the option you said can work too!